include/ola/gen_callbacks.py

```diff
@@ -41,14 +42,15 @@ def Header():
    *
    * You should have received a copy of the GNU Lesser General Public
    * License along with this library; if not, write to the Free Software
-   * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+   * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 \
```

Comment:
The other option here is to take this one line out of the triple quotes and print it separately, if we'd rather avoid the backslash.